types/errors_test.go

```diff
@@ -50,8 +48,11 @@ func (s testErrorSuite) TestError(c *C) {
 		ErrInvalidWeekModeFormat,
 		ErrWrongValue,
 	}
-	for _, err := range kvErrs {
-		code := terror.ToSQLError(err).Code
-		c.Assert(code != mysql.ErrUnknown && code == uint16(err.Code()), IsTrue, Commentf("err: %v", err))
-	}
+
+	t.Run("ValidateErrorCodes", func(t *testing.T) {
```

Comment:
You probably should do:

```
for _, err := range kvErrs {
     t.Run("ValidateErrorCodes", func(t *testing.T) {
			code := terror.ToSQLError(err).Code
			require.Equal(t, uint16(err.Code()), code, fmt.Sprintf("err: %v", err))
  })
}
```

But my recommendation would be:

```
func TestError(t *testing.T) {
t.Parallel()
```

There is no need to parallel tests for such a small test

Reply:
Done, PTAL